Scripts/extract_release_notes.rb

```diff
@@ -0,0 +1,81 @@
+# Parses the release notes file to extract the current version information and
+# puts it to STDOUT.
+#
+# To update the release notes for localization:
+#
+# ruby ./this_script >| Simplenote/Resources/release_notes.txt
+#
+# To generate the App Store Connect release message:
+#
+# ruby ./this_script | pbcopy
+#
+# To generate the GitHub and App Center release message:
+#
+# ruby ./this_script -k | pbcopy
```

Comment:
I'm sure there are many improvements we could make on this script. Eventually, it would be great for the release toolkit to actually do it, but until each project has a different convention for the release notes formatting, that might not be feasible.

The current iteration does the job for what I need to do right now.